stubs/attrs/METADATA.toml

```diff
@@ -1,2 +1,3 @@
-version = "0.1"
+version = "19.1"
```

Comment:
Doesn't this mean these stubs are for attrs 19? They are for attrs 18 and possibly older.

Reply:
According to `git log`, they were last updated for 19.2, but I assume they will still work with 19.1.